File: ddd-application/src/main/java/victor/training/ddd/agile/domain/repo/SprintItemRepo.java
package victor.training.ddd.agile.domain.repo;

import victor.training.ddd.agile.common.CustomJpaRepository;
import victor.training.ddd.agile.domain.model.BacklogItem;
import victor.training.ddd.agile.domain.model.SprintItem;

public interface SprintItemRepo extends CustomJpaRepository<SprintItem, Long> {
}

File: ddd-application/src/main/java/victor/training/ddd/agile/repo/SprintItemRepo.kt
package victor.training.ddd.agile.repo

import victor.training.ddd.agile.common.CustomJpaRepository
import victor.training.ddd.agile.entity.BacklogItem
import victor.training.ddd.agile.entity.SprintItem

interface SprintItemRepo : CustomJpaRepository<SprintItem  , Long>